.github/workflows/ci.yml

```diff
@@ -0,0 +1,155 @@
+on:
+  push:
+    branches:
+      - develop
+      - epic/**
+      - release/**
+      - integration-libs/**
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+env:
+  CYPRESS_KEY: ${{ secrets.CYPRESS_KEY }}
+
+name: Spartacus CI pipeline
+jobs:
+  no_retries: 
+    name: Verify retries
+    runs-on: ubuntu-latest
+    steps:
+      - name: Force fail CI. No retries allowed. Please push an empty commit.
+        uses: actions/github-script@v5
+        with:
+          script: |
+            core.setFailed('Please push an empty commit - git commit --allow-empty -m "Trigger Build"') 
+        if: ${{ github.run_attempt > 1 }}
+  unit_tests_core:
+    needs: no_retries
+    name: Unit tests for core Spartacus libs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
Can we extract node version to a github variable?

Reply:
Changed